moses/moses/main/populate_atomspace.cc

```diff
@@ -0,0 +1,72 @@
+#include <moses/data/table/table.h>
+#include <moses/data/table/table_io.h>
+#include <opencog/atomspace/AtomSpace.h>
+#include <moses/utils/value_key.h>
+#include <opencog/atoms/base/Link.h>
+#include "populate_atomspace.h"
+#include <opencog/atoms/value/Value.h>
+
+namespace opencog {
+namespace moses {
+
+	void populate(AtomSpace *as, ITable &itable) {
```

Comment:
Actually even that indentation can be scrapped, i.e. the entire namespace block require no indentation. See that file as example https://github.com/singnet/as-moses/blob/master/moses/combo/combo/vertex.h#L54

Reply:
`#include <moses/data/table/table.h>
#include <moses/data/table/table_io.h>
#include <opencog/atomspace/AtomSpace.h>
#include <moses/utils/value_key.h>
#include <opencog/atoms/base/Link.h>
#include "populate_atomspace.h"
#include <opencog/atoms/value/Value.h>

namespace opencog { namespace moses {

void populate(AtomSpace *as, ITable &itable) {
	std::vector<multi_type_seq>::const_iterator it, end;
	int col_size = itable.get_types().size();
	for (int i = 0; i < col_size; i++) {
		id::type_node col_type = itable.get_types().at(i);
		int row_size = itable.size();
		vertex_seq col = itable.get_column_data(i);
		switch (col_type) {
			case id::boolean_type: {
				std::vector<ValuePtr> col_values = {};
				Handle feature = createNode(SCHEMA_NODE, itable.get_labels().at(i));
				for `

Reply:
you mean like no tab at all for the void function but the inside code be tabed probably once/twice.

Reply:
Yes, I think you understood right.